onchain/cairo/launchpad/src/launchpad/locker/lock_manager.cairo

```diff
@@ -297,27 +272,27 @@ pub mod LockManager {
         }
 
         fn user_locks_length(self: @ContractState, user: ContractAddress) -> u32 {
-            let user_locks: Vec<felt252> = self.user_locks.read(user);
-            user_locks.len
+            let user_locks_length = self.user_locks_list.entry(user).len();
+
+            user_locks_length.try_into().unwrap()
         }
 
         fn user_lock_at(
             self: @ContractState, user: ContractAddress, index: u32
         ) -> ContractAddress {
-            let user_locks: Vec<felt252> = self.user_locks.read(user);
-            user_locks[index]
+            self.user_locks_list.entry(user).at(index.into()).read()
```

Comment:
Please let's use the user_locks, because this variable are never used.

Reply:
How can we use `user_locks` with an index when there is no `Vec` or `List` being used.